viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -121,35 +122,52 @@
 
 
     var userFilterVisibility = function (showFilters) {
+      var searchBox = $('#userFilterSpan');
+      var textField = $('#userNameBox');
       if (showFilters) {
-        $('#userFilterSpan').show();
-        if ($('#userNameBox').val() === "") {
-          $('#userNameBox').val(applicationModel.getSessionUser());
+        searchBox.show();
+        if (textField.val() === "") {
+          textField.val(applicationModel.getSessionUser());
         }
-        filterByUser();
       } else {
-        $('#userFilterSpan').hide();
+        textField.val("");
+        searchBox.hide();
       }
+      filterByUser();
     };
 
     function bindEvents() {
 
       eventbus.once('roadAddressProjects:fetched', function(projects) {
-        projectArray = projects;
-        createProjectList(projects);
+        projectArray = _.filter(projects, function(proj) {
+          return proj.statusCode !== projectStatus.Deleted.value; //filter deleted projects out
+        });
+        createProjectList(projectArray);
       });
 
-      var createProjectList = function(projects) {
+      var createProjectList = function(projects, sortFunction = function(a,b) {return a.ely - b.ely}, order = 1) {
```

Comment:
i got one error in compilation where it seems to warning that in current jshint the default parameters  (either functins or values) dont work.